Assets/GothicVR/Scripts/UI/ImageController.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using UnityEngine;
+using GVR.Caches;
+using GVR.Util;
+using GVR.Phoenix.Interface;
+using TMPro;
+
+public class ImageController : MonoBehaviour
```

Comment:
This can be renamed to UI Texture Manager or something like that

Reply:
Resolved.